src/appservice/bot/ListCommand.tsx

```diff
@@ -77,9 +77,10 @@ const restart = defineInterfaceCommand<AppserviceBaseExecutor>({
         {
             name: "mjolnir",
             acceptor: findPresentationType("UserID"),
+            description: 'The userid of the mjolnir to restart'
         }
     ]),
-    command: async(context, _keywords, mjolnirId: UserID): Promise<CommandResult<true>> => {
+    command: async (context, _keywords, mjolnirId: UserID): Promise<CommandResult<true>> => {
```

Comment:
Did your editor put this whitespace here? Worried that this isn't done anywhere else in the code base (and yes it sucks we have no proper linter+prettier to just not worry about stuff.)

Not a major issue though.

Reply:
I guess that was the editor yeah. Vscode did that. I thought I avoided them but apparently I didnt.